app/publisher.py

```diff
@@ -12,7 +12,7 @@ class DurableTopicPublisher:
 
     EXCHANGE = os.getenv("SEFT_RABBIT_EXCHANGE", 'message')
     PUBLISH_INTERVAL = 1
-    ROUTING_KEY = "JWT"
+    ROUTING_KEY = os.getenv("SDX_SEFT_PUBLISHER_ROUTING_KEY")
```

Comment:
It then defaults to the queue name, which is what the adapter and the consumer use anyway